test/features/currency/currency_dropdown_test.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:didpay/features/currency/currency_dropdown.dart';
+import 'package:didpay/features/payment/payment_amount_state.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import '../../helpers/test_data.dart';
+import '../../helpers/widget_helpers.dart';
+
+void main() async {
+  await TestData.initializeDids();
+
+  late ValueNotifier<PaymentAmountState?> mockState;
+  const testCurrency = 'USD';
+
+  setUp(() {
+    mockState = ValueNotifier<PaymentAmountState?>(
+      PaymentAmountState(
+        offeringsMap: TestData.getOfferingsMap(),
+      ),
+    );
+  });
```

Comment:
there's no need to use `late` and `setUp()` here, you can just create a `ValueNotifier` in `currencyDropdownTestWidget()`